packages/e2e-tests/tests/plain-email-tests.spec.ts

```diff
@@ -88,3 +93,39 @@ test('check "New E-Mail" option is shown and a chat can be created', async () =>
     .filter({ hasText: subject })
   await expect(chatListItem).toBeVisible()
 })
+
+test('group can be created but no non key contacts can be added', async () => {
+  const userA = existingProfiles[0]
+  const userB = existingProfiles[1]
+  const subject = 'Test Chat Subject'
+  const emailUserB = userB.address
+  // prepare last open chat for receiving user
+  await switchToProfile(page, userA.id)
+  await page.locator('#new-chat-button').click()
+
+  const createGroupButton = page.getByRole('button', { name: 'New Group' })
+
+  await expect(createGroupButton).toBeVisible()
+
+  await createGroupButton.click()
+
+  await page.getByTestId('group-name-input').fill(subject)
+  await page.locator('#addmember').click()
+
+  await page.getByTestId('add-member-search').fill(emailUserB)
+
+  const contactList = page.getByTestId('add-member-dialog').locator('li button')
+
+  await expect(contactList).toHaveCount(0)
+
+  await expect(page.getByTestId('ok')).toHaveAttribute('disabled')
```

Comment:
This can potentially break, due to search taking some time.

Reply:
What do you mean? The search for the ok button attribute?

Reply:
Well, verifying that the list is empty. It could be empty at first, and then become populated.